internal/api/v1/routes/routes.go

```diff
@@ -18,15 +18,17 @@ func RegisterRoutes(
 	// Instances endpoints
 	instances := v1.Group("/instances")
 	instances.Get("/", instanceHandler.ListInstances).Name("ListInstances")
-	instances.Post("/", instanceHandler.CreateInstance).Name("CreateInstance")
-	instances.Delete("/", instanceHandler.DeleteInstance).Name("DeleteInstance")
+	// TODO: Will add CreateInstance and DeleteInstance endpoints to modify an existing job
+	// instances.Post("/", instanceHandler.CreateInstance).Name("CreateInstance")
+	// instances.Delete("/", instanceHandler.DeleteInstance).Name("DeleteInstance")
```

Comment:
on principle I don't like merging PRs that break main like this. 
create and delete instance is a foundational workflow. Not having them on main will impact basically all other work.

Reply:
Create and delete instance is done by job right now there should be no endpoint directly for dealing with instances at the moment. in future we might add those endpoints for some cases.